lib/surface/components/form/range_input.ex

```diff
@@ -0,0 +1,77 @@
+defmodule Surface.Components.Form.RangeInput do
+  @moduledoc """
+  Generates a text input.
+
+  Provides a wrapper for Phoenix.HTML.Form's `range_input/3` function.
+
+  All options passed via `opts` will be sent to `range_input/3`, `value` and
+  `class` can be set directly and will override anything in `opts`.
+
+
+  ## Examples
+
+  ```
+  <RangeInput form="volume" field="percent" min: "0" max: "100"/>
+  ```
+  """
+
+  use Surface.Component
+
+  import Phoenix.HTML.Form, only: [range_input: 3]
+  import Surface.Components.Form.Utils
+
+  alias Surface.Components.Form, warn: false
```

Comment:
You shouldn't need `warn: false` here since `Form` is being used in `context get`. In case you remove it and you start receiving warnings, please let me know, it's a bug.